google/cloud/forseti/services/inventory/base/gcp.py

```diff
@@ -2228,6 +2228,94 @@ def iter_kms_keyrings(self, project_id, location=None):
         raise ResourceNotSupported('Key Management Service is not supported by '
                                    'this API client')
 
+    def iter_kubernetes_nodes(self, project_id, zone, cluster):
+        """Iterate k8s nodes in an organization from GCP API.
```

Comment:
please update all the docstring below in other methods to reflect the proper parent, instead of organization.

Reply:
Done